src/render/src/App.svelte

```diff
@@ -272,20 +339,30 @@ function makeZoomable() {
 onMount(() => {
   makeZoomable();
 });
-</script>
 
+</script>
 <div class="controlsContainer">
   <div class="controls">
     <button onclick={resetView}>Reset View</button>
     <button
       onclick={openCode}
       disabled={!Boolean(codeUrl)}
       title={Boolean(codeUrl) ? "" : "Only available for GitHub code"}
-      >Open Code</button
-    >
+    >Open Code</button>
     <button onclick={saveSVG}>Download SVG</button>
   </div>
+  {#if rawSVG}
```

Comment:
This _works_, but hinges on an unrelated implementation detail. 
A better way to do this would be to have the metadata object as a potentially `undefined` variable, and checking that.

Also - how are we handling `graph` rendering, vs `github`?